### ISSUE TITLE
Feature Request: Show date of Magic Sealed Data

### ISSUE THREAD

**Issue by tooomm** [dr4fters/dr4ft#1151]
## Detailed Description
`...and Magic Sealed Data booster rules commit d6db322` should have a date as well:
`...and Magic Sealed Data booster rules commit d6db322 (2020-07-08)`

**Comment by HerveH44** [comment]
@ZeldaZach booster rules are now part of MTGJson v5 right ? We could stop fetching the files from magic sealed and just read the data from mtgjson files ?

**Comment by ZeldaZach** [comment]
Yep that's correct. Booster rules are apart of v5 sets under `booster`

**Comment by tooomm** [comment]
> Yep that's correct. Booster rules are apart of v5 sets under `booster`

Does MTGJSON use taw's "Magic Sealed Data" data as source as well?
And how frequent is the `booster` information updated?
